internal/app/server/open_saves.go

```diff
@@ -78,10 +78,11 @@ func newOpenSavesServer(ctx context.Context, cfg *config.ServiceConfig) (*openSa
 		}
 		cache := cache.New(redis.NewRedisWithConfig(&cfg.RedisConfig), &cfg.CacheConfig)
 		server := &openSavesServer{
-			cloud:      cfg.ServerConfig.Cloud,
-			blobStore:  gcs,
-			metaDB:     metadb,
-			cacheStore: cache,
+			cloud:             cfg.ServerConfig.Cloud,
+			blobStore:         gcs,
+			metaDB:            metadb,
+			cacheStore:        cache,
+			maxInlineBlobSize: cfg.BlobConfig.MaxInlineSize,
```

Comment:
Maybe can we pass cfg itself to openSavesServer instead of individual fields? That'll make future changes easier.

Reply:
That's much better! I'll try to change it like so.